site/docs/guide/filter-queries.md

```diff
@@ -9,7 +9,7 @@ Other filtering of messages is left to the developer, which often leads to endle
 
 On the contrary, **grammY ships with its own query language** that you can use in order to **filter for exactly the messages** you want.
 
-This allows for over 820 different filters to be used, and we may add more over time.
+This allows for over 1150 different filters to be used, and we may add more over time.
```

Comment:
Is 1150 an estimate or the exact count?

Reply:
The exact count is 1153, but I don't want to give the exact number in case we have to do minor changes to the implementation. I don't want to maintain it that closely.

Reply:
LGTM then. Writing 1100 would look better imo, but it is not critical.